### PR TITLE
Fix cannot read private member #isReadyPromise and dependencies update

### PR DESCRIPTION
**Pull request by isSerge**
- update dependencies
- fix cannot read private member #isReadyPromise
- adjust TS and Eslint

### CODE REVIEW

File: src/lib/client.ts [hunk 1]
```diff
@@ -26,24 +26,27 @@ const SUNIT = 1000000000000000000n
 const NETWORK_RPC = process.env.PUBLIC_API_WS || "ws://localhost:9944"
 const LOCAL_RPC = process.env.LOCAL_API_WS || "ws://localhost:9944"
 const appsLink = "https://polkadot.js.org/apps/?rpc=" + NETWORK_RPC + "#/explorer/query/"
+
+const publicApi = new ApiPromise({
```
Comment (nazar-pc) [inline src/lib/client.ts:30]: Why not doing this in constructor, so you have control over when it happens instead of running on file inclusion?

Comment (isSerge) [reply src/lib/client.ts:30]: I am afraid that is the actual fix for the problem. Originally API was assigned directly as a class property and it was causing the problem. Apparently, Quasar or Vue use `Proxy` somewhere under the hood - https://github.com/polkadot-js/api/issues/4656 Using constructor does not solve this.

Comment (nazar-pc) [reply src/lib/client.ts:30]: You need to use async contructor and wait for API to actually initialize before you use it, did you try that to make sure it is actually working?

If contructors are broken with Quasar or Vue then bad things will be literally everywhere, I don't believe that is the case, but async initialization of Polkadot.js's API might easily be.

Comment (isSerge) [reply src/lib/client.ts:30]: Async constructor may work, but it will require some refactoring (not just Client class), this was intended as a quick fix

Comment (nazar-pc) [reply src/lib/client.ts:30]: Can you at least not refactor things to use global variable and assign property of the class to the constant? And TODO there to refactor to async constructor.

Comment (isSerge) [reply src/lib/client.ts]: I made an attempt to fix this with as minimal effort as possible, but it requires much more. Please, check

File: src/lib/client.ts [hunk 2]
```diff
@@ -1,45 +1,44 @@
...
-  protected localApi: ApiPromise = new ApiPromise({
-    provider: new WsProvider(LOCAL_RPC, false),
-    types: util.apiTypes
-  })
   protected farmed: FarmedBlock[] = []
   protected clearTauriDestroy: event.UnlistenFn = () => {}
   protected unsubscribe: event.UnlistenFn = () => {}
+
+  localApi: ApiPromise;
+  publicApi: ApiPromise;
```
Comment (nazar-pc) [inline src/lib/client.ts:35]: Should these be public?

File: tsconfig.json [hunk 3]
```diff
@@ -4,7 +4,7 @@
   "compilerOptions": {
     "target": "ESNext",
     "strict": true,
-    "baseUrl": ".",
+    "baseUrl": "./src",
```
Comment (nazar-pc) [inline tsconfig.json:7]: I always wonder why is it `./src` and not just `src` :slightly_smiling_face:

**Review by nazar-pc — APPROVED** [review]
Generally looks better than initial attempt, but I am convinced over and over again that the architecture here isn't very reliable with all this global state.

**Comment by isSerge** [comment]
@nazar-pc global state is usually a store like redux or vuex (similar for vue), never saw a state like this

File: src/lib/client.ts (hunk 2 above)
Comment (isSerge) [reply src/lib/client.ts:35]: currently yes

**Review by nazar-pc — APPROVED** [review]

**Pull request merged**